cacheops/lua/invalidate.lua

```diff
@@ -1,7 +1,11 @@
 local prefix = KEYS[1]
 local db_table = ARGV[1]
 local obj = cjson.decode(ARGV[2])
-
+local conj_del_fn = 'unlink'
+-- If Redis version < 4.0 we can't use UNLINK
+-- TOSTRIP
+conj_del_fn = 'del'
+-- /TOSTRIP
```

Comment:
So here's the hack: set the redis function name to use to `unlink`, then set it to `del` instead – but delete that code from the script if Redis >= v4.0.

Reply:
Magic 🌟 🌟 🌟 🌟 🌟